src/bitcoinrpc.cpp

```diff
@@ -295,6 +295,7 @@ static const CRPCCommand vRPCCommands[] =
     { "decodescript",           &decodescript,           false,  false },
     { "signrawtransaction",     &signrawtransaction,     false,  false },
     { "sendrawtransaction",     &sendrawtransaction,     false,  false },
+    {"burn",                    &burn,                   false,  false },
```

Comment:
Are you sure you got these flags right?

Reply:
looks right. id like a space thou where it is missing XD.

Reply:
I figured it has to be the same as for sending.